openquakeplatform/exposure/views.py

```diff
@@ -199,33 +202,39 @@ def export_exposure(request):
             return response
     else:
         occupancy = 0  # 'residential' by default
-    data = util._get_currency_and_taxonomy_name(sr_id, occupancy)
-    if data:
-        currency, taxonomy_name, taxonomy_version = data
-    else:
-        # No records were found, corresponding to the given sr_id and occupancy
-        # ==> we return a 204 (the server successfully processed the request,
-        #                      but is not returning any content)
-        #     because it means there's no study corresponding to
-        #     the given combination of parameters
-        response = HttpResponse(status="204")
-        return response
+        
     output_type = request.GET.get('output_type')
     if not output_type or output_type not in ('csv', 'nrml'):
         msg = ('Please provide the parameter "output_type".'
                ' Available choices are "csv" and "nrml".')
         response = HttpResponse(msg, status="400")
         return response
+
+    content_disp_type = ('attachment' if 'geddb' in connections else 'inline')
```

Comment:
With the new `moon` probably we can return to use `attachment` as in master.

Reply:
Fixed here: https://github.com/gem/oq-platform2/pull/16/commits/aebefdb84cc50dedcf77bcf1473c17f03c3cd0fb

Reply:
Fixed here: https://github.com/gem/oq-platform2/pull/16/commits/218f9c64361907fb13bb83ea8b11c6901cca11b2